### PR TITLE
Prepare repo to build 2.2.1

### PR DESCRIPTION
**Pull request by natemcmaster**
Changes:

* Update dependencies to 2.2.0 rtm
* Update branding to 2.2.1
* Update package baselines to 2.2.0
* Add a restore source for 2.2.0 RTM

FYI @mmitche

### CODE REVIEW

File: eng/Sources.props [hunk 1]
```diff
@@ -3,6 +3,13 @@
 
   <PropertyGroup Label="RestoreSources">
     <RestoreSources>$(DotNetRestoreSources)</RestoreSources>
+
+    <!-- This feed contains the 2.2.0 RTM runtime. This can be removed after it is released to nuget.org. -->
```
Comment (muratg) [inline eng/Sources.props]: Should we add a post-release activity to clean up things like this?

Comment (mmitche) [reply eng/Sources.props]: I don't think this is necessary.  The dotnet-core blob and myget feeds contain the 2.2.0 packages

File: eng/tools/BaselineGenerator/Program.cs [hunk 2]
```diff
@@ -78,11 +80,15 @@ private async Task<int> Run()
 
                 if (!File.Exists(nupkgPath))
                 {
-                    var url = $"{source}/{id}/{version}";
+                    var url = _feedv3.HasValue()
+                        ? $"{source}/{id.ToLowerInvariant()}/{version}/{id.ToLowerInvariant()}.{version}.nupkg"
```
Comment (muratg) [inline eng/tools/BaselineGenerator/Program.cs]: :/ I'm sure this will bite us elsewhere too.

File: eng/Sources.props (hunk 1 above)
Comment (natemcmaster) [reply eng/Sources.props]: Yeah, I'll open a bug on aspnetcore-internal to track this.

Comment (natemcmaster) [reply eng/Sources.props]: > The dotnet-core blob and myget feeds contain the 2.2.0 packages

Cool, i'll remove this feed now. Didn't see this comment before I merged.

**Pull request merged**